lib/src/main/java/ca/barrenechea/widget/recyclerview/decoration/StickyHeaderDecoration.java

```diff
@@ -140,7 +153,7 @@ private int getHeaderTop(RecyclerView parent, View child, View header, int adapt
                     long nextId = mAdapter.getHeaderId(adapterPosHere);
                     if (nextId != currentId) {
                         final View next = parent.getChildAt(i);
-                        final int offset = getAnimatedTop(next) - (header.getHeight() + getHeader(parent, adapterPosHere).itemView.getHeight());
```

Comment:
from what I understand, getY() is just getTop() + getTranslationY(). But i see no problem changing it back.